packages/code-du-travail-frontend/src/support/__tests__/__snapshots__/Support.test.js.snap

```diff
@@ -55,7 +55,7 @@ exports[`<Support /> should render 1`] = `
               viewBox="0 0 20 20"
             >
               <title>
-                Recherche par adresse
+                Rechercher par adresse
```

Comment:
can be a separate PR :smiling_imp: 
(don't do it if you're tired of git. It's ok if you leave it here)

Reply:
I couldn't resist ;)

Reply:
So are you leaving it here ?

Reply:
yes